src/utils/vc/crypto/custom-linkeddatasignature.js

```diff
@@ -122,6 +161,12 @@ export default class CustomLinkedDataSignature extends jsigs.suites.LinkedDataSi
       const signatureBytes = await getSigBytes(verifyData);
       finalProof.proofValue = CustomLinkedDataSignature.encodeProofValue(signatureBytes);
     } else {
+      if (!this.alg) {
+        throw new Error('Suite doesn\'t contain required alg parameter');
+      }
+
+      // TODO: use signJWS
```

Comment:
Whats the reason for not calling `signJWS`  here but in `issueCredential`?

Reply:
its an old todo actually, isnt needed here anymore (as signing is done by other crypto methods). `signJWS` is called in `issueCredential` because JWT VCs dont go through the linked data signature suite

Reply:
ive also moved signJWS/createJWS into their own file as they ideally shouldnt be here